createDataset.py

```diff
@@ -25,162 +151,74 @@ def getWhatsAppData():
                     myMessage = cleanMessage(myMessage)
                     responseDictionary[otherPersonsMessage.rstrip()] = myMessage.rstrip()
                     otherPersonsMessage, myMessage = "",""
-                otherPersonsMessage = otherPersonsMessage + str(row['Content']) + " "
```

Comment:
What's the reason for this again? Is it a python 2 -> 3 thing?

Reply:
The diff is being rendered wrong, that `str` cast is in line 157, `getLinkedinData`. It didn't have an `str` cast originally but that might be an oversight. I'll add it real quick